android/app/src/main/res/values/strings.xml

```diff
@@ -662,7 +662,8 @@
 	<string name="bookmarks_empty_list_title">This list is empty</string>
 	<string name="bookmarks_empty_list_message">To add a bookmark, tap a place on the map and then tap the star icon</string>
 	<string name="category_desc_more">…more</string>
-	<string name="export_file">Export file</string>
+	<string name="export_file">Export KML/KMZ file</string>
```

Comment:
This should be in the regenerated commit to avoid conflicts.

Reply:
Fixed

Reply:
Looks like strings.xml is still modified here. Can it be moved to another commit?